src/data/faq_de.json

```
@@ -376,6 +379,216 @@
+                                    "Wenn Sie manuell über die neue Funktion \"Tests erfassen und andere warnen\" andere CWA-Anwender*innen direkt warnen, wird zwar kein Test in der App registriert, aber im Tagebuch dokumentiert, dass Sie Andere gewarnt haben.",
+                                    "Siehe auch: <a href='#warn_without_tan_how_to' target='_blank'>Wie kann ich andere Nutzer warnen, wenn ich positiv getestet worden bin?</a>"
+                                ]
+                            },
+                            {
+                                "title": "Kann ich für einen Selbsttest ein offizielles digitales Zertifikat erhalten?",
+                                "anchor": "warn_without_tan_self_test_certificate",
+                                "active": false,
+                                "textblock": [
+                                    "Nein, europaweit sind für Selbsttests keine digitalen Zertifikate vorgesehen. Testzertifikaten können nur für beaufsichtigte PCR-Tests (ausgenommen PoC-NAT PCR-Tests) und beaufsichtigte Schnelltests ausgestellt werden."
+                                ]
+                            },
+                            {
+                                "title": "Kann ich für einen registrierten positiven Testbefund als Krankschreibung nutzen?",
+                                "anchor": "warn_without_tan_sickness_certificate",
+                                "active": false,
+                                "textblock": [
+                                    "Nein, die Warnung anderer mit der neuen Funktion \"Tests erfassen und andere warnen\" führt gar nicht zur Anzeige eines positiven Testergebnisses in der Corona-Warn-App. Abgesehen davon ist die Anzeige in der CWA arbeitsrechtlich nicht zur Krankschreibung bzw. Arbeits-/Schulbefreiung zugelassen. Wenden Sie sich bitte an den Arzt Ihres Vertrauens oder an das zuständige Gesundheitsamt."
+                                ]
+                            },
+                            {
+                                "title": "Wie lange bleibt der Test in der App zu sehen?",
+                                "anchor": "warn_without_tan_in_app_display",
+                                "active": false,
+                                "textblock": [
+                                    "Die Warnung anderer mit der neuen Funktion \"Tests erfassen und andere warnen\" führt gar nicht zur Anzeige eines positiven Testergebnisses in der Corona-Warn-App.",
+                                    "Für Tests, die wie bisher in der App registriert werden, gilt nach wie vor: Tests stellen in der Diagnostik von Infektionskrankheiten immer nur eine Momentaufnahme dar, sie sind daher nicht unbegrenzt lange gültig. Nach 72 Stunden wird daher ein registrierter Test in der Corona-Warn-App als „veraltet“ grau dargestellt."

```

1. Remove "grey"
2. Change "dargestellt" to "angezeigt" (sounds better IMO)
```suggestion
                                    "Für Tests, die wie bisher in der App registriert werden, gilt nach wie vor: Tests stellen in der Diagnostik von Infektionskrankheiten immer nur eine Momentaufnahme dar, sie sind daher nicht unbegrenzt lange gültig. Nach 72 Stunden wird daher ein registrierter Test in der Corona-Warn-App als „veraltet“ angezeigt."
```

**IMPORTANT:** Please cross-check the information in this FAQ entry, I have something in the back of my mind which tells me that only RATs are displayed as out of date after **48h** and PCR tests are only displayed as out of date after **21 days**.

I'm sure @mlenkeit knows what I mean and can shine a light on that?

---

Your question has been forwarded and we will try to come back to you shortly, please stand by

---

@larswmh Has my suggestion been implemented anyway or will we wait for an answer to the question?

---

@Ein-Tim your suggestion has been implemented anyway

---

Ok, thanks! Please still leave this conversation open until we have an answer.

---

It would be nice to get this question answered before this PR is merged! I will nevertheless approve the PR now, as everything else looks good to me! 

@larswmh

---

@Ein-Tim

your concerns were valid. For negative RATs, it is actually 48 hours. After reviewing it with the team, we decided to remove that part of the article, since the first part already answers the question. https://github.com/corona-warn-app/cwa-website/pull/3322/commits/0412615801db415e94b4bfd2a7e08b361add4634

---

Sounds good, this conversation can be marked as resolved.